pkg/virt-controller/watch/workload-updater/workload-updater_test.go

```diff
@@ -577,7 +573,7 @@ func newKubeVirt(expectedNumOutdated int) *v1.KubeVirt {
 	}
 }
 
-func newVirtualMachine(name string, isMigratable bool, image string, vmiIndexer cache.Indexer, podStore cache.Store) *v1.VirtualMachineInstance {
+func newVirtualMachine(name string, isMigratable bool, image string, vmiIndexer cache.Store, podStore cache.Store) *v1.VirtualMachineInstance {
```

Comment:
nit: `podStore cache.Store` can be `podIndexer cache.Indexer`

Reply:
will do

Reply:
Yeah, I thought the same after a while.